.circleci/npmjs-publish.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -e
set -o pipefail
```

I think you want these at a minimum (possibly `set -u` as well), but I'm not sure if `set -e` will work because of https://github.com/koalaman/shellcheck/wiki/SC2155. ¯\_(ツ)_/¯